File: Exemplos M10/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Bridge/SetSenhaServer.cs
namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Bridge
{
    class SetSenhaServer : BridgeCommand
    {
        readonly string senha;
        readonly bool habilitada;

        public SetSenhaServer(string senha, bool habilitada) : base("SetSenhaServer")
        {
            this.senha = senha;
            this.habilitada = habilitada;
        }

        protected override string FunctionParameters()
        {
            return "\"senha\"" + ":" + "\"" + senha + "\"" + "," +
                    "\"habilitada\"" + ":" + habilitada;
        }
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/BRIDGE/Commands/SetSenhaServer.ts
import { BridgeCommand } from "../BridgeCommand";

export class SetSenhaServer extends BridgeCommand {
  readonly senha: string;
  readonly habilitada: boolean;

  constructor(senha: string, habilitada: boolean) {
    super("SetSenhaServer");
    this.senha = senha;
    this.habilitada = habilitada;
  }

  functionParameters(): object {
    return { senha: this.senha, habilitada: this.habilitada };
  }
}
